src/digest/digest.rs

```diff
@@ -454,6 +459,17 @@ pub extern fn SHA512_4(out: *mut u8, out_len: c::size_t,
     polyfill::slice::fill_from_slice(out, digest);
 }
 
+#[cfg(feature="no_asm")]
+unsafe fn sha256_block_data_order(_: &mut [u64; MAX_CHAINING_LEN / 8], _: *const u8, _: c::size_t){
+    unimplemented!();
+}
+
+#[cfg(feature="no_asm")]
+unsafe fn sha512_block_data_order(_: &mut [u64; MAX_CHAINING_LEN / 8], _: *const u8, _: c::size_t){
+    unimplemented!();
```

Comment:
Let's drop these definitions that call `unimplemented!()`. I'll keep this commit separate from the one that adds the Rust implementations of these functions, but I'll land them together. I'd rather have unbuilding code in the intermediate state between those two commits than building-but-broken code.